database/migrations/2023_01_30_101610_create_logs_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use dnj\ErrorTracker\Contracts\LogLevel;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('logs', function (Blueprint $table) {
+            $table->id();
+            $table->enum('level', getEnumValues(LogLevel::cases()));
```

Comment:
technically it must use/save enum values not names.
but it's up to [eloquent enum cast](https://laravel.com/docs/9.x/eloquent-mutators#enum-casting). check it's behavior.